assets/charts/control-plane/pod-checkpointer/templates/daemonset.yaml

```diff
@@ -74,3 +73,6 @@ spec:
       - name: var-run
         hostPath:
           path: /var/run
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Isn't this a default as value says? Do we need to set this explicitly?

Reply:
No it is not unfortunately the default at k8s level. 

```
$ kubectl explain pod.spec.securityContext.seccompProfile
...
   type <string> -required-
     type indicates which kind of seccomp profile will be applied. Valid options
     are:

     Localhost - a profile defined in a file on the node should be used.
     RuntimeDefault - the container runtime default profile should be used.
     Unconfined - no profile should be applied.
```

Right now it is automatically converted from annotation to the field by apiserver.

Reply:
Or I guess PSP controller mutates the pod to set it?

Reply:
Yes I think so.

Reply:
So, I understand it does not work then, if we remove it?

Reply:
PSP controller was mutating wrongly IIUC. Since our PSPs always had the seccomp annotations and even though pods had default seccomp profile annoatation as `docker/default` the pods were still failing. So something has changed.

I couldn't find if docker/default was disabled and now runtime/default is the supported one.